firefox-ios/Client/Coordinators/Browser/BrowserNavigationHandler.swift

```diff
@@ -7,6 +7,7 @@ import Storage
 import WebKit
 
 import struct MozillaAppServices.CreditCard
+import enum MozillaAppServices.VisitType
```

Comment:
not a fan of declaring these types from another service, I prefer to have a wrapper object. However, to minimize changing will keep the method for `VisitType` the same